addons/mail/models/mail_thread.py

```diff
@@ -1409,6 +1409,9 @@ def _message_parse_extract_payload(self, message, save_original=False):
                     alternative = True
                 if part.get_content_type() == 'multipart/mixed':
                     mixed = True
+                if part.get_content_type() == 'bin/plain' and '.pdf' in part.get_filename():
```

Comment:
```suggestion
                if part.get_content_type() == 'bin/plain' and 'part.get_filename(failobj='').endswith('.pdf'):
```

Reply:
@reth-odoo  thank you for the remarks, updated.